try:
    print("what")
except Exception as e:
    print(e)


